modules/igv/1.0/config/default.yaml

```diff
@@ -0,0 +1,118 @@
+lcr-modules:
+    
+    igv:
+
+        inputs:
+            # Available wildcards: {unix_group} {seq_type} {tumour_sample_id} {normal_sample_id} {pair_status} {genome_build}
+            maf: "__UPDATE__"
+
+        regions: 
+            # Provide regions files as lists in their respective genome builds so that liftover of coordinates occurs properly
+            oncodriveclustl:
+                grch37: []
+                hg38: []
+            hotmaps:
+                grch37: []
+                hg38: []
+            bed:
+                grch37: []
+                hg38: []
+            maf:
+                grch37: []
+                hg38: []
+            mutation_id:
```

Comment:
Can we add here the example formatting of what is expected in that file? Does it have to have a header and expects certain column names?

Reply:
Done ! I added what column and format is required for mutation_id file format